### DISCUSSION TITLE
Possible to remove strict code checking for a single file

### DISCUSSION THREAD

**Discussion by anthodb** [BowlerHatLLC/vscode-as3mxml#582] [Q&A] [answered]
Hello, thanks for this superb extension!
I was looking in the doc for a possible way to disable "strict" code checking for a single file (old library working but not strict at all).
Something like "ts-ignore" that would stop the code checking for the file, or at least the "strict" code checking?
Do you have some kind of option for that? 
Thanks!

**Comment by joshtynjala** [comment, marked as answer]
No, the compiler doesn't have any sort of option like that. The code intelligence in vscode-as3mxml uses the compiler internally, so it doesn't have that feature either.

**Reply by anthodb** [reply]
ok thanks
